eq-author-api/middleware/identification/verifyJwtToken.test.js

```diff
@@ -0,0 +1,76 @@
+process.env.REACT_APP_FIREBASE_PROJECT_ID = "TEST_FIREBASE_ID";
+
+const jwt = require("jsonwebtoken");
+const fs = require("fs");
+const verifyJwtToken = require("./verifyJwtToken");
+const {
+  VERIFY_VALIDITY_OPTIONS,
+} = require("../../constants/googleServiceKeys");
+
+const mockKid = "MY_SECRET_KEY";
+const mockUserId = "MOCK_USER_ID";
+
+const mockCertPrivate = fs.readFileSync(
+  __dirname + "/__mock__/rsa-private.pem"
+);
+const mockCertPublic = fs.readFileSync(__dirname + "/__mock__/rsa-public.pem");
```

Comment:
Can this just be `./__mock__/rsa-public.pem`?

Reply:
Apparently not:
`ENOENT: no such file or directory, open './__mock__/rsa-private.pem'`

Reply:
Good point. I always forget this.